shell/common/gin_converters/blink_converter.cc

```diff
@@ -379,6 +402,28 @@ bool Converter<blink::WebMouseEvent>::FromV8(v8::Isolate* isolate,
   return true;
 }
 
+v8::Local<v8::Value> Converter<blink::WebMouseEvent>::ToV8(
+    v8::Isolate* isolate,
+    const blink::WebMouseEvent& in) {
+  auto dict = gin_helper::Dictionary::CreateEmpty(isolate);
+
+  dict.Set("type", in.GetType());
+  dict.Set("clickCount", in.ClickCount());
+  dict.Set("movementX", in.movement_x);
+  dict.Set("movementY", in.movement_y);
+  dict.Set("button", in.button);
+
+  gfx::PointF position_in_screen = in.PositionInScreen();
+  dict.Set("x", position_in_screen.x());
+  dict.Set("y", position_in_screen.y());
+
+  gfx::PointF position_in_widget = in.PositionInWidget();
+  dict.Set("globalX", position_in_widget.x());
+  dict.Set("globalY", position_in_widget.y());
+
```

Comment:
I would have expected `globalX`/`globalY` to be the mouse position on screen, and `x`/`y` to be the position  within the web contents. Are they intentionally reversed here?

Reply:
nope, typo - fixed